### DISCUSSION TITLE
"null" on an integer field

### DISCUSSION THREAD

**Discussion by nervusdm** [directus/directus#15004] [Archive]
Hey guys,

Using api, sending a "NULL" data, I have an error message, pretty simple : 

(poids means weight)

```
 [errors] => Array
        (
            [0] => Array
                (
                    [message] => "poids" must be a number
                    [extensions] => Array
                        (
                            [code] => FAILED_VALIDATION
                            [field] => poids
                        )

                )

        )

)
```

I don't get it. null is allowed.

Any idea ?

![Capture d’écran 2022-08-10 à 12 41 27](https://user-images.githubusercontent.com/4217180/183882256-cb9ef937-0415-46be-a75a-f883fb61650e.png)

**Comment by rijkvanzanten** [comment]
Do you have any validation rules configured?

**Reply by nervusdm** [reply]
Nope,
So for now, I just do : If "is null" then remove it..
Thanks for your answer

**Reply by rijkvanzanten** [reply]
How are you sending the null data? Are you sending `"null"` instead of `null` by chance?
